server/src/test/java/org/elasticsearch/search/SearchServiceTests.java

```diff
@@ -1027,13 +1027,13 @@ public void testCanMatch() throws Exception {
 
         CountDownLatch latch = new CountDownLatch(1);
         SearchShardTask task = new SearchShardTask(123L, "", "", "", null, Collections.emptyMap());
-        assertEquals(8, numWrapInvocations.get());
+        assertEquals(6, numWrapInvocations.get());
```

Comment:
I had to change these values because we early exit due to shard filtering and the counter is not incremented.